src/components/Toggle/Toggle.style.css

```diff
@@ -0,0 +1 @@
+@import "../../index.css";
```

Comment:
If this file `src/components/Toggle/Toggle.style.css` is not used anywhere, please remove it.